src/pages/Home.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Heading } from '@chakra-ui/react';
+import Carousel from '../components/Product/Carousel';
+import Products from '../components/Product/Products';
+
+export default function Home() {
+    const [products, setProducts] = useState([]);
```

Comment:
Good point, but in `/src/pages/Products.js`, the `products` in a similar `useState` is used for two components, so the state is declared outside of the component in `src/components/Product/Products.js`. Instead of creating another component, I stuck with this component and similarly declared the state outside of the component.

Reply:
Merge conflict resolved!